### PR TITLE
adding Altdentifier presence and a small fix on Carl-bot presence

### PR DESCRIPTION
**Pull request by NMW03**
image
image
image
image
image

### CODE REVIEW

File: Altdentifier/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    "service": "Altdentifier",
+    "description": {
+      "en": "The best Discord security and checkpoint bot!"
+    },
+    "url": [
+      "altdentifier.com"
+    ],
+    "version": "1.0",
+    "logo": "https://altdentifier.com/static/cop.png",
```
Comment (Bas950) [inline Altdentifier/dist/metadata.json]: Please upload their logo to another site yourself. Imgur or your is-inside.me

Comment (NMW03) [reply Altdentifier/dist/metadata.json]: Done

File: Carl-bot/dist/presence.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ presence.on("UpdateData", () => {
         presenceData.startTimestamp = browsingStamp;
         if (document.location.pathname.includes("/dashboard/")) {
             presenceData.details = "Managing the settings of";
-            title = document.getElementById('__BVID__16__BV_button_').innerText
+            title = document.querySelector("#__BVID__17__BV_button_ > strong").innerText
```
Comment (Bas950) [inline Carl-bot/dist/presence.js]: update version in metadata, and upload proof that it works.

**Comment by NMW03** [comment]
image

**Pull request merged**